crates/autopilot/src/infra/persistence/dto/order.rs

```diff
@@ -260,3 +264,43 @@ impl From<boundary::EcdsaSignature> for domain::auction::order::EcdsaSignature {
         }
     }
 }
+
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum FeePolicy {
```

Comment:
What do you think about using a model here that can actually be written straight into the db (which I believe this enum cannot)?

Reply:
Should be shared probably, so I will link this to be tackled in https://github.com/cowprotocol/services/pull/2118 once this PR is merged.

Reply:
mentioned https://github.com/cowprotocol/services/pull/2118#issuecomment-1883224848